services/cocoapods/cocoapods-platform.service.js

```diff
@@ -19,15 +19,26 @@ module.exports = class CocoapodsPlatform extends LegacyService {
       {
         title: 'Cocoapods platforms',
         namedParams: { spec: 'AFNetworking' },
-        staticPreview: {
-          label: 'platform',
-          message: 'ios | osx | watchos | tvos',
-          color: 'lightgrey',
-        },
+        staticPreview: this.render({ platforms: 'ios | osx | watchos | tvos' }),
       },
     ]
   }
 
-  // Legacy route handler is defined in cocoapods.service.js.
-  static registerLegacyRouteHandler() {}
+  static render({ platforms }) {
+    return {
+      message: platforms,
+      // https://github.com/badges/shields/pull/184
+      color: '#989898',
+    }
+  }
+
+  async handle({ spec }) {
+    const data = await this.fetch({ spec })
+    const platforms = Object.keys(data.platforms).join(' | ')
```

Comment:
Nit: The `.join(' | ')` probably belongs in `render`. Passing `Object.keys(data.platforms)` seems slightly better.